v_4_0_0/cloudconfig_test.go

```diff
@@ -63,70 +75,3 @@ func TestCloudConfig(t *testing.T) {
 		}
 	}
 }
-
-func TestCloudConfigTemplating(t *testing.T) {
```

Comment:
I removed this test completely as files are rendered as base64 strings. Rendering itself is verified by other tests, so the question is do we want to check that those exact templates are still rendered into proper configuration?

Reply:
No. Those tests weren't worth much IMO.